docs/BBSR_recipe_testlist.md

```diff
@@ -15,14 +15,14 @@ Refer to [UEFI SCT Test Case Specification](https://uefi-sct-testcasespec.readth
 </tr>
 <tr>
 <td rowspan=32 colspan=1>🔲 RuntimeServicesTest</td>
-<td rowspan=3 colspan=1>✅ SecureBootTest</td>
-<td rowspan=1 colspan=1>✅ ImageLoading</td>
+<td rowspan=3 colspan=1>✅ SecureBootTest<sup>[arm]</sup></td>
```

Comment:
The entire  SecureBootTest suite will be upstreamed soon. They will no longer be Arm specific